src/components/button/index.module.css

```diff
@@ -17,6 +17,15 @@
   }
 }
 
+.textLink {
+  display: inline;
+  padding: 0;
```

Comment:
I don't see the needed of `padding: 0; ` and `background: transparent;`. We already have it from link and reset

Reply:
Update!